CAM Platform/workspaces.md

```diff
@@ -13,812 +13,892 @@ Manage and perform actions on workspaces.
+| [GET /services/workspaces/{workspace_id}/boxes](#get-servicesworkspacesworkspace_idboxes) | Gets the boxes in a workspace |
+| [GET /services/workspaces/{workspace_id}/instances](#get-servicesworkspacesworkspace_idinstances) | Gets the instances in a workspace |
 
-* 200
 
-**Error Response Codes**
+## GET /services/workspaces
+Gets the default workspace if no parameter is provided. If ids parameter (a list of workspace ids) is provided then it fetches all the workspaces corresponding to that list.
 
-* Bad Request (400)
+### URL
 
-**Request**
+#### Structure
+```
+[GET] /services/workspaces
+```
+#### Example
+```
+[GET] https://cam.ctl.io/services/workspaces
+```
+### Request
 
+#### Headers
 ```
 Content-Type: application/json
 Authorization: Bearer your_json_web_token
 ElasticBox-Release: 4.0
 ```
+#### URI Parameters
+| NAME | TYPE | DESCRIPTION | REQ. |
+|------| ---- | ----------- | ---- |
+| ids | string | Fetches the workspaces corresponding to each workspace id in the string. The ids will be separated by commas. For example: workspace4,operations,workspace5. | No |
+
+#### Request body parameters
+
+* None
+
+### Response
+#### Normal Response Codes
+
+- **202** Accepted
+
+#### Common Error Response Codes
 
-**Response Parameters**
+- **400** Bad Request
+- **401** Unauthorized
+- **404** Not Found
+
+#### Response Parameters
 
 | Parameter | Type |Description |
 |-----------|------|------------|
-| organizations | array | Lists of organizations of the workspace. |
-| updated | string | Date of the last update. Example “2015-07-02 10:23:47.748740” |
-| name | string | Workspace name. |
-| created | string | Creation date. Example “2015-07-02 10:23:47.748740” |
-| uri | string | Workspace uri. |
-| members | array | Lists members of a team workspace. |
+| add_provider | boolean | Indicates true if a personal workspace has a provider |
+| billing_notice | boolean | If the flag is False we don't show the billing notice to the user |
+| clc_alias | string | (optional) Account Id for billing customers that have an account at CL |
+| clc_username | string | If user has CLC authentication |
+| created | string | Creation date |
+| costcenter | string | Cost center id |
+| dashboard_notice | boolean | If the flag is False we don't show the dashboard notice to the user |
+| deleted | boolean | If the workspace has been deleted |
+| deploy_instance | boolean | Indicates if the workspace has deployed an instance |
+| email | string | User email |
+| email_validated_at | string | Email validation date. |
+| favourites | array | List of user's favourite workspaces. User set a workspace as favourite when he clicks on a star. |
 | group_dns | array | List of fully qualified names of LDAP groups to which a user’s personal workspace belongs. You can’t update this field. Present in Personal Workspaces |
+| icon | string | Workspace icon |
```

Comment:
It should be stated that it's a relative URL to a file that is stored as a blob in CAM.